src/main/java/com/haxwell/apps/quizki/controllers/QuestionController.java

```diff
@@ -32,4 +35,11 @@ public QuestionController (QuestionService qs) {
 	public CreatedQuestionDTO create(@RequestBody @Valid CreateQuestionDTO cqDTO) throws CreateQuestionDTOException {
 		 return qs.createQuestion(cqDTO);
 	}
+	
+	@GetMapping(value = "/{id}")
```

Comment:
For consistency, we should add a `@GetMapping` to the `create` method also, even though it has a default. It will just make the code easier to read. Consistency.

Reply:
Not sure what you mean here. On line 33 there is a `@PostMapping` annotation for the `create` method. It's not a `GET` method. All the methods are using the base URL that's mapped by the `@RequestMapping` on the class on line 23. The `create` method uses just the  base URL. The `getQuestionById` method has to add the `/12` to specify the question id. Maybe that's the difference you're referring to?

Reply:
Stupid GitHub didn't include lines above 35 in my view... and I just now noticed, after reading your comment. Soo... never mind what I was talking about here..